scripts/update-all-metadata/go.mod

```diff
@@ -0,0 +1,5 @@
+module example.com/m
```

Comment:
It's not a good module name, what about `github.com/paketo-buildpacks/dep-server/scripts/update-all-metadata`